front/app/modules/commercial/smart_groups/containers/RulesGroupFormWithValidation/RulesGroupForm.tsx

```diff
@@ -27,68 +35,97 @@ const SSectionField = styled(SectionField)`
 import { Multiloc } from 'typings';
 import { TRule } from '../../components/UserFilterConditions/rules';
 
-export interface Props {}
 export interface RulesFormValues {
   rules: TRule[];
   title_multiloc: Multiloc;
-  membership_type: 'rules';
+  membership_type: 'rules' | 'manual';
```

Comment:
The component is only capable of dealing of groups with `rules` membership_type (this means it's a smart group), so I think the previous typing was more exact. Maybe you needed this to get rid of difficult to solve typing issues?